core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -85,10 +86,13 @@ public void stopAndRemoveContainer(String containerId, String imageName) {
     }
 
     private void stopContainer(String containerId, String imageName) {
-        boolean running;
+        boolean present;
         try {
-            InspectContainerResponse containerInfo = dockerClient.inspectContainerCmd(containerId).exec();
-            running = containerInfo.getState().getRunning();
+            Optional<Container> container = dockerClient.listContainersCmd().exec()
```

Comment:
actually...... do we even have to check here? Force removal should do the job anyway